examples/carrots.py

```diff
@@ -1,29 +1,29 @@
 # important to note that python's float type has the same precision as the C++ double
 
-import pydp as dp # our privacy library
+import pydp as dp    # our privacy library
 import pandas as pd
 
-class CarrotReporter:
+class CarrotReporter:  # Creating a class ClassReporter
 
     data_filename: str
     epsilon: float
 
-    def __init__(self, data_filename, epsilon):
+    def __init__(self, data_filename, epsilon): #Function to read the csv file and creating a dataframe
```

Comment:
can the comments be written at the top of each function rather in the same line?

Reply:
yeah sure